kernel/include/priority_q.h

```diff
@@ -15,6 +15,12 @@ extern int32_t z_sched_prio_cmp(struct k_thread *thread_1,
 
 bool z_priq_rb_lessthan(struct rbnode *a, struct rbnode *b);
 
+#if defined(CONFIG_64BIT)
+#define NBITS 64
+#else
+#define NBITS 32
```

Comment:
Since this is a header (albeit a kernel-internal one), probably best to take the opportunity to rename this "_SCHED_MQ_NBITS" or something similarly namespaced.